daemon/execdriver/lxc/driver.go

```diff
@@ -418,13 +427,13 @@ func cgroupPaths(containerID string) (map[string]string, error) {
 	logrus.Debugf("subsystems: %s", subsystems)
 	paths := make(map[string]string)
 	for _, subsystem := range subsystems {
-		cgroupRoot, cgroupDir, err := findCgroupRootAndDir(subsystem)
-		logrus.Debugf("cgroup path %s %s", cgroupRoot, cgroupDir)
+		cgroupRoot, _, err := findCgroupRootAndDir(subsystem)
+		logrus.Debugf("cgroup path %s /", cgroupRoot)
```

Comment:
seems like /proc/self/mountinfo contains the wrong mount point for cgroups (inside dind with lxc execdriver they are /docker/CID while lxc just needs / and then it uses lxc/CID on root) maybe cause lxc inherits the mnt namespaces mounts (just guessing)

ping @LK4D4